src/test/java/com/google/step/snippet/W3SchoolClientTest.java

```diff
@@ -0,0 +1,60 @@
+package com.google.step.snippet;
+
+import com.google.step.snippet.data.Card;
+import com.google.step.snippet.external.W3SchoolClient;
+import org.junit.Assert;
```

Comment:
By convention, we statically import junit assert methods

e.g. `import static org.junit.Assert.assertEquals;`